packages/design-tokens/README.md

```diff
@@ -0,0 +1,36 @@
+# Kaizen Tokens
+Design tokens for all platforms.
+
+## About
+Design Tokens are the heart of every Design System. The tokens represented here are platform-agnostic (JSON), as this will help us contribute to & facilitate the maintenance of living style guides. This package defines all the option tokens in Kaizen. 
```

Comment:
```suggestion
Design Tokens are the heart of every Design System. The tokens represented here are platform-agnostic (JSON), as this will help us contribute to & facilitate the maintenance of living style guides. This package defines all the option tokens in Kaizen.

(An "option token" is a token like `$kz-color-wisteria-800: #35374A` that is one of the options in a palette and contains no information about _how_ it should be used, as opposed to a "decision token" like `$default-text-color: $kz-color-wisteria-800` which does information about how the token should be used.)
```

Reply:
Optional suggestion here for an explanation of what an option token is – I figured it couldn't hurt to spread awareness.